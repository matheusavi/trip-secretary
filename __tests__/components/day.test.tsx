import { render } from "@testing-library/react";
import Day from "../../app/components/day";

it("renders day unchanged", () => {
  const { container } = render(<Day day={1} />);
  expect(container).toMatchSnapshot();
});
