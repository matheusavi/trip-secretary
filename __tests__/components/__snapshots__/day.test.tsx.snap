// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders day unchanged 1`] = `
<div>
  <div
    class="bg-white col-span-1 col-start-1 col-end-1 border-r-2 text-xs"
    data-testid="hour-1"
  >
    01:00
  </div>
</div>
`;
